HW45_DB_repository/HW45_DB_repository.py

```diff
@@ -0,0 +1,73 @@
+import sqlite3
+
+
+class Teacher:
+
+    def __init__(self, name, subject, experience_years, salary):
+        self.name = name
+        self.subject = str(subject).capitalize()
+        self.experience_years = experience_years
+        self.salary = salary
+
+    def __repr__(self):
+        return f"Teacher({self.name}, {self.subject}, {self.experience_years}, {self.salary})"
+
+
+class TeachersRepository:
+
+    def __init__(self, db_path):
+        self._connection = sqlite3.connect(db_path, isolation_level=None)
+        self._cursor = self._connection.cursor()
+        self._cursor_execute = self._cursor.execute("CREATE TABLE IF NOT EXISTS Teachers(id INTEGER PRIMARY KEY, name TEXT, subject TEXT, experience_years INTEGER, salary REAL);")
```

Comment:
What the reason of saving into field? execute() returns the same cursor object (self._cursor)